### DISCUSSION TITLE
i am trying to set kvrocks cluster in kubernetes

### DISCUSSION THREAD

**Discussion by Goutami-Gowda** [apache/kvrocks#1169] [Q&A]
i am using this docker image: apache:kvrocks ; and i am trying to set a cluster using this document. https://kvrocks.apache.org/docs/Cluster/kvrocks-cluster-introduction. 
 but i am not able to find this command.  CLUSTERX SETNODES
  can some one help me

**Comment by git-hulk** [comment]
Hi @Goutami-Gowda 

Cloud you paste your commands and the error message?

**Reply by Goutami-Gowda** [reply]
command :  CLUSTERX SETNODES 
error: 
bash: CLUSTERX: command not found
<img width="575" alt="image" src="https://user-images.githubusercontent.com/63769299/206448427-392c2e98-9027-425c-b796-edf51182ce32.png">

**Reply by Goutami-Gowda** [reply]
what is the binary file path for CLUSTERX SETNODES in apache:kvrocks image?
